lib/repo-data.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export const repoPairs = [{
+  spec: 'https://github.com/w3c/vc-di-ecdsa',
+  testSuite: 'https://github.com/w3c/vc-di-ecdsa-test-suite',
```

Comment:
I do also wonder about renaming these--or even just making these arrays--so others could (eventually) use this code to compare repos (at least A vs. B) that aren't "spec" nor "testSuite".